_schemaV2.graphql

```diff
@@ -2858,6 +2858,23 @@ type BackupSecondFactors {
 # A list of backup second factors or errors
 union BackupSecondFactorsOrErrorsUnion = BackupSecondFactors | Errors
 
+type BankAccount {
+  # Name on the bank account
+  accountHolderName: String
+
+  # Bank name
+  bankName: String!
+
+  # A globally unique ID.
+  id: ID!
+
+  # A type-specific ID likely used as a database ID.
+  internalID: ID!
+
+  # Last four characters of the account identifier
+  last4: String!
```

Comment:
I only included a minimal set of bank account data. This is what we currently have available from gravity v1: 
```
{
  "id": "13039038-68c9-44ef-973c-ab6f269646e0",
  "customer_account_id": "",
  "external_id": "pm_1L88BJGK3Gnpfa3OC421s4sv",
  "account_holder_name": "",
  "account_holder_type": "individual",
  "bank_name": "STRIPE TEST BANK",
  "country": "",
  "currency": "",
  "routing_number": "110000000",
  "last4": "6789",
  "fingerprint": "Qf0RvlKZ2z3hh89Z",
  "status": "",
  "created_at": "2022-06-07T19:35:16.203Z",
  "updated_at": "2022-06-07T19:35:16.203Z",
  "one_time_use": false,
  "user_id": "57da9676a09a6720300017cb",
  "account_type": "checking",
  "deactivated_at": null
}
```